examples/sample-demos/openebs-zfs-deploy/volumesnapshotclass.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: snapshot.storage.k8s.io/v1
```

Comment:
This folder name is apt for the demo?!

Reply:
> This folder name is apt for the demo?!

yes, openebs-zfs will be reused in different use cases for volume part. so its deployment is kept separate